playbooks/utils/replace_vm_host.yml

```diff
@@ -0,0 +1,75 @@
+---
+#  This playbook will delete a VM and replace it. This is the only warning you will get. Running this playbook has destructive consequences
+#     
+- name: replace a host on VSphere
+  hosts: localhost
+  remote_user: pulsys
+  become: true
+  vars_files:
+    - ../../group_vars/vsphere/vault.yml
+    - ../../group_vars/vsphere/{{ runtime_env | default('staging') }}.yml
+
+  tasks:
+
+    - name: Gather MAC address of VM to replace
+      community.vmware.vmware_vm_info:
+        hostname: '{{ vcenter_hostname }}'
+        username: '{{ vcenter_username }}'
+        password: '{{ vcenter_password }}'
+        vm_name: "{{ vm_to_replace }}"
+      facts:
+        - mac_address
+        - uuid
+        - datastore_url
+      register: vm_info
```

Comment:
Can we print this info? If something errors after deletion such that the new VM isn't created, I think we would lose these values.

Reply:
Addressed in b8753ad8d70698d4e8ce093d9ccbc4c4fe330bce